core/arch/arm/mm/mobj.c

```diff
@@ -155,12 +155,6 @@ struct mobj *mobj_phys_alloc(paddr_t pa, size_t size, uint32_t cattr,
 	enum teecore_memtypes area_type;
 	void *va;
 
-	if ((pa & CORE_MMU_USER_PARAM_MASK) ||
-	    (size & CORE_MMU_USER_PARAM_MASK)) {
-		DMSG("Expect %#x alignment", CORE_MMU_USER_PARAM_SIZE);
-		return NULL;
-	}
-
```

Comment:
Because I use mobj_phys to represent shared buffers. They are not always page-aligned.